pkg/enterprisetypes/installer.go

```diff
@@ -0,0 +1,6 @@
+package enterprisetypes
+
+type Installer struct {
+	ID   string
+	Yaml string
```

Comment:
Does gofmt want to name this YAML instead?

Reply:
nope